javascript/src/main/components/ChannelMessages/SearchResultsView.js

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import {useAuth0} from "@auth0/auth0-react";
+import useSWR from "swr";
+import {fetchWithToken} from "main/utils/fetch";
+import ToolkitProvider, { Search } from 'react-bootstrap-table2-toolkit';
+import TimeFormatter from "./time"
+const { SearchBar } = Search;
+
+const GetMessageContents = (text, slackUsers) => {
+    return text.replace(/<@([A-Z0-9]{11})>/g, (_,userId) => {
+        if(slackUsers != null) {
+            for(let i = 0; i < slackUsers.length; i++) {
+                if(slackUsers[i].id === userId) {
+                    return "@" + slackUsers[i].real_name;
+                }
+            }
+        }
+        return "@" + userId;
+    })
+}
+
+const formatBracketedText = (text) => {
+    var bracketRegEx = /<(.*?)>/g;
+    var found = text.match(bracketRegEx)
+    if (found){
+        for (let i=0; i<found.length; i++){
+            var current = found[i].replace('<', '');
+            current = current.replace('>', '');
+            if (found[i].includes("|") && (found[i].includes("mailto") || found[i].includes("http") || found[i].includes("tel"))){      // embedded links
+                var links = current.split('|');
```

Comment:
This is also copied straight from `MessageListView.js`.  `SearchResultsView.js` is simply `MessageListView.js` with the addition of a channel column using `channelFormatter`. I think this should also be a separate issue in general about the use of `let` and `const` over `var`.

Reply:
Sure, but let's not replicate bad code if we can instead fix it.
It may also be a good idea to put in a story about refactoring this code out so we don't have to repeat it everywhere.